src/views/MainPage.vue

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable vue/no-parsing-error */
+<template>
+  <v-container>
+      <div v-for="film in films" :key="film.id" class="card-film">
+      <div class="card-film__img">
+        <img :src="'https://image.tmdb.org/t/p/w500' + film.poster_path" alt="poster">
```

Comment:
old fashioned strings. try to use new one `/``
Also I think there should be full path of the images and not relative